ops/delivery-e2e.cloudbuild.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+steps:
+  # Clean up any old Emblem instance cruft.
+  - name: '${_REGION}-docker.pkg.dev/${_PROJECT}/e2e-deployer/e2e-deployer:${_E2E_DEPLOYER_SHA}'
+    entrypoint: /bin/bash
+    args:
+      - ./terraform/cleanup.sh
+    env:
+      - "EMBLEM_BILLING_ACCOUNT=${_EMBLEM_BILLING_ACCOUNT}"
+      - "EMBLEM_ORGANIZATION=${_EMBLEM_ORGANIZATION}"
+      - "PROJECT_NUMBER=$PROJECT_NUMBER"
+
+      # Project IDs
+      - "PROD_PROJECT=${_PROD_PROJECT}"
+      - "STAGE_PROJECT=${_STAGE_PROJECT}"
+      - "OPS_PROJECT=${_OPS_PROJECT}"
```

Comment:
Delivery testing is not quite "in the demo" for Emblem, so we can merge this but I'm seeing some signs that it may create complications. Possibly instead of spreading the pieces around the repo, we should put together a dedicated top-level directory?

Reply:
Those env vars are no longer needed; they're only used in the "ephemeral" `clean_project_setup.sh` script.

Reply:
Hm. Some changes (e.g. Terraform) need to be made in other locations, so this isolation wouldn't be perfect.

I'm not _entirely_ against making delivery testing a "fast follow" (particularly if there's more pressing stuff to work on), **but** I'd argue the "peace of mind" we get from having it is well worth it.

(This is true even if it's something we only run manually - personally, I appreciate having automated tests for Terraform changes!)

Reply:
I'm okay having it be part of things, but only if the delivery testing adds low cognitive overhead and no additional user setup requirements.

We could have a directory like:

```
installation-testing/
  - terraform
  - cloudbuild
  - scripts
```

To aggressively isolate. I know we didn't discuss this before, sorry to make this concept a surprise. Is it an achievable pivot?

Reply:
I'm on board with it, but can we do it in a separate PR? (for "separation of concerns" reasons)